src/main/java/thaumicboots/api/IBoots.java

```diff
@@ -80,6 +88,17 @@ default boolean changeOmniState(ItemStack stack) {
         return omni;
     }
 
+    default boolean changeInertiaCancellingState(ItemStack stack) {
```

Comment:
I know you felt the naming convention was awkward, but mind changing it to "isIntertiaCanceled" ?

Just to maintain consistency with the rest, I mean the reason why they were called that in the first place were booleans, and this is a boolean so.